src/main/scala/com/martinsnyder/chatserver/InputMessage.scala

```diff
@@ -29,19 +29,19 @@ object InputMessage {
   // Parses a string into a command
   def parse(user: String, text: String): InputMessage =
     splitFirstTwoWords(text) match {
-      case ("/help", _, _) => Help(user)
-      case ("/room", "", "") => EnterRoom(user, DefaultRoomName)
+      case ("/help", _, _)     => Help(user)
+      case ("/room", "", "")   => EnterRoom(user, DefaultRoomName)
       case ("/room", room, "") => EnterRoom(user, room.toLowerCase)
-      case ("/room", _, _) => InvalidInput(user, "/room takes a single, optional argument")
-      case ("/rooms", _, _) => ListRooms(user)
-      case ("/members", _, _) => ListMembers(user)
-      case (cmd, _, _) if cmd.startsWith("/") => InvalidInput(user, s"unknown command - $cmd")
-      case _ => Chat(user, text)
+      case ("/room", _, _)     => InvalidInput(user, "/room takes a single, optional argument")
+      case ("/rooms", _, _)    => ListRooms(user)
+      case ("/members", _, _)  => ListMembers(user)
+      case (s"/$cmd", _, _)    => InvalidInput(user, s"unknown command - $cmd")
```

Comment:
Slick - was this automatic or did you make this improvement by hand?

Reply:
Yes.
It is a new feature of scala 2.13. 
I just read another article, which mentioned this new capability and wanted to try it out ;-)